src/libpmem2/deep_sync_linux.c

```diff
@@ -23,31 +23,26 @@
  * on given region_id
  */
 int
-pmem2_deep_sync_write(int region_id)
+pmem2_deep_sync_write(unsigned region_id)
 {
 	LOG(3, "region_id %d", region_id);
 
 	char deep_flush_path[PATH_MAX];
 	int deep_flush_fd;
 
 	if (util_snprintf(deep_flush_path, PATH_MAX,
-		"/sys/bus/nd/devices/region%d/deep_flush", region_id) < 0) {
+		"/sys/bus/nd/devices/region%u/deep_flush", region_id) < 0) {
 		ERR("!snprintf");
 		return PMEM2_E_ERRNO;
 	}
 
 	if ((deep_flush_fd = os_open(deep_flush_path, O_WRONLY)) < 0) {
-		ERR("!os_open(\"%s\", O_WRONLY)", deep_flush_path);
-		return PMEM2_E_NOSUPP;
+		LOG(1, "Cannot open deep_flush file %s", deep_flush_path);
+		return 0;
 	}
 
```

Comment:
if we are changing this, you should also read from this file to check its contents as @kilobyte described here: #4567